stable/traefik/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.7.0
-appVersion: 1.3.1
+version: 1.8.0
```

Comment:
sure thing. I wasn't fully aware of the logic behind the chart `Version`. I agree that it should follow the version of the image. e.g. Major upgrade of `appVersion`  will mean a major upgrade of `version`